docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md

```diff
@@ -76,3 +76,4 @@ Here is a table of the currently supported networks on HyperRPC and their respec
 
 
 
+
```

Comment:
The number of empty lines increases with every PR :)

Reply:
Will fix, something must be wrong with the script.